### DISCUSSION TITLE
Adding a global FUNDING.yml shouldn't add sponsor link to forked repos

### DISCUSSION THREAD

**Discussion by simonw** [community/community#13717] [Other Feature Feedback, Questions, & Ideas]
I added a global `FUNDING.yml` file here, to put my sponsorship info on all of my repos at once: https://github.com/simonw/.github/blob/main/FUNDING.yml

I just noticed that it's no showing the "Sponsor" CTA on the repos that I have forked in order to send PRs as well!

<img width="1160" alt="image" src="https://user-images.githubusercontent.com/9599/160466298-c733012d-497e-4433-ae73-e19ec40484ee.png">

This is bad: I don't want to make it look like I'm trying to profit from the work of others.

I can turn off that sponsorship logo on a per-repo basis by going into settings and unchecking the box:

<img width="835" alt="image" src="https://user-images.githubusercontent.com/9599/160466486-13f8baa0-13d0-4014-99a8-58c077440a51.png">

But I have 190 forked repos and I fork more all the time, so it's frustrating to have to remember to do this.

**Comment by jlord** [comment]
Hi! That makes sense as lot of forks aren't maintained forks of a project, though some might be (and reasonable to sponsor development on). This is good feedback and perhaps we could have a global setting to turn this default behavior off.
